run.cmd

```diff
@@ -0,0 +1,40 @@
+@if not defined _echo @echo off
+setlocal
+
+if not defined VisualStudioVersion (
+  if defined VS140COMNTOOLS (
+    call "%VS140COMNTOOLS%\VsDevCmd.bat"
+    goto :Run
+  )
+  echo Error: Visual Studio 2015 required.
+  echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
+  exit /b 1
+)
+
+:Run
+:: Clear the 'Platform' env variable for this session, as it's a per-project setting within the build, and
+:: misleading value (such as 'MCD' in HP PCs) may lead to build breakage (issue: #69).
+set Platform=
+set TOOLRUNTIME_DIR=%~dp0Tools
+set BOOTSTRAP_URL=https://raw.githubusercontent.com/dotnet/buildtools/master/bootstrap/bootstrap.ps1
```

Comment:
Downloading a powershell script, that has code to download more stuff, that has code to download more stuff ... I does not look like an improvement to me. I do not see what was wrong with the existing init-tools.cmd script. The existing init-tools.cmd script was easy to tweak as necessary to make progress ... it will be much harder to do with this system.